scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -43,6 +43,14 @@ workflow GvsExtractCallset {
           scatter_count = scatter_count
     }
 
+    call GetLastModifiedDateOrUniqueValue {
+        input:
+           data_project = data_project,
+           default_dataset = default_dataset,
+           service_account_json = service_account_json,
+           dataset_table = "alt_allele"
```

Comment:
is there some notion of a constant in WDL that would make sense for this to point to?

Reply:
I made `dataset_table` an input in case it could be used by other GVS WDL calls to take advantage of call-caching.  If this doesn't end up being the case, I can hardcode it.

Reply:
If I'm understanding this correctly, this should check the same table(s) used to drive the extract (not alt_allele) -- specifically the inputs to `ExtractTask` for `fq_cohort_extract_table` and `fq_samples_to_extract_table`.  

If this task could take a list of fully qualified table names (so they don't have to be in the same project or dataset) and get the latest modified date) that would probably do it